.eslintrc.json

```diff
@@ -5,13 +5,29 @@
     "standard",
     "prettier",
     "prettier/standard",
+    "plugin:unicorn/recommended",
     "plugin:ava/recommended",
     "plugin:you-dont-need-lodash-underscore/all"
   ],
   "rules": {
     "no-process-exit": 0,
     "object-shorthand": "error",
     "require-await": "error",
+    "unicorn/custom-error-definition": 2,
+    "unicorn/no-unused-properties": 2,
+    "unicorn/filename-case": [2, { "case": "kebabCase", "ignore": [".*.md"] }],
+    "unicorn/explicit-length-check": [2, { "non-zero": "not-equal" }],
+    "unicorn/no-process-exit": 0,
+    "unicorn/no-null": 0,
```

Comment:
That's a contentious one. I personally agree with this rule, but I think this might be too controversial to be useful. [See rule documentation](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-null.md). Also, this would require lots of refactoring.

Reply:
I think it's good for consistency .